CHANGELOG.md

```diff
@@ -1,5 +1,473 @@
 # Changelog
 
+All notable changes to this project will be documented in this file.
+
+## [unreleased]
+
+### Features
+
+- *(uncategorized)* Add weak constraints to make rects closer to each other in size ✨ ([#395](https://github.com/ratatui-org/ratatui/issues/395))
+([6153371](https://github.com/ratatui-org/ratatui/commit/61533712be57f3921217a905618b319975f90330))
+
+  ````text
+  Also make `Max` and `Min` constraints MEDIUM strength for higher priority over equal chunks
+  ````
```

Comment:
Is it possible to make these text codeblocks a html details tag?

```
  <details>
  <summary>Show more</summary>
  Also make `Max` and `Min` constraints MEDIUM strength for higher priority over equal chunks
  <details>
```

would render as 

  <details>
  <summary>Show more</summary>
  Also make `Max` and `Min` constraints MEDIUM strength for higher priority over equal chunks
  </details>
  
---

If it is possible, is that something we might want to do?

Reply:
I looked at that as an option first. I couldn't get it to render in lists yesterday, but having a new look, I think indent fixes that.

The bigger problem I see with this is that it means that you can't search for text in the non-expanded regions - which limits the use case of being able to easily find when something specific changed.

Perhaps a nice way to have both would be to have a high level overview up top that links to more details of each commit underneath, but that might be overkill for now.